documentation/project/contributing/prerequisites.md

```diff
@@ -37,6 +69,46 @@ export JAVA_HOME="$(/usr/libexec/java_home -v 17)"
 Consider allocating not less than 4GB of memory for your docker.
 Otherwise, some apps within a stack (e.g. `kafka-ui.yaml`) might crash.
 
+To check how much memory is allocated to Docker:
+
+```
+docker info
+```
+
+You will find the total memory and used memory in the output. if you won't find used memory thats mean memory limits are not set for containers.
+
+#### To allocate 4GB of memory for Docker on Ubuntu
+
+1. Open the Docker configuration file in a text editor using the following command:
+
+```
+sudo nano /etc/default/docker
```

Comment:
that's applicable only for linux, this file is present neither on windows nor macos